types/zingchart/index.d.ts

```diff
@@ -4,4 +4,6 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.3
 // Minimum TypeScript Version: 3.7
-export * from './es6';
+import { zingchart } from './es6';
+
+export default zingchart;
```

Comment:
This is not correct. This file must correspond to https://unpkg.com/browse/zingchart@2.8.9/index.js, which is a CJS module, not an ES module. It should be `export = zingchart`, and users can use a default import by enabling `esModuleInterop` in their own projects. In the future, TypeScript will support separate definitions for CJS and ESM importers, but for now, you need to model the `main` entrypoint in this file.